client/src/components/home/HomeComponent.tsx

```diff
@@ -61,59 +38,83 @@ export const HomeComponent = (props: HomeComponentProps): JSX.Element => {
-            <LearnMoreItem
-              headerText={'Azure Communication Services virtual appointments'}
-              headerLink={'https://learn.microsoft.com/azure/communication-services/tutorials/virtual-visits'}
-              description={
-                'This tutorial describes concepts for virtual appointment applications. After completing this tutorial and the associated Sample Builder, you will understand common use cases that a virtual appointment delivers...'
-              }
-            />
-            <LearnMoreItem
-              headerText={'Azure Communication Services Rooms (Preview)'}
-              headerLink={'https://learn.microsoft.com/en-us/azure/communication-services/concepts/rooms/room-concept'}
-              description={
-                'Azure Communication Services provides a concept of a room for developers who are building structured conversations such as virtual appointments or virtual events. Rooms currently allow voice and video calling...'
-              }
-            />
-            <LearnMoreItem
-              headerText={'Get Started with UI Library'}
-              headerLink={
-                'https://learn.microsoft.com/azure/communication-services/quickstarts/ui-library/get-started-composites?tabs=kotlin&pivots=platform-web'
-              }
-              description={
-                'Get Started with Azure Communication Services UI Library to quickly integrate communication experiences into your applications. In this quickstart, learn how to integrate UI Library composites into an application and set up...'
-              }
-            />
+            <LearnMore />
           </Stack>
         </Stack>
       </Stack>
     </Stack>
   );
 };
+
+const HomeButtons = ({ theme, setError }): JSX.Element => {
+  const callCreateRoom = async (): Promise<void> => {
+    try {
+      const redirectUrl = await createRoomAndRedirectUrl();
+      window.location.assign(redirectUrl);
+    } catch (error) {
+      setError(error);
+    }
+  };
+
+  return (
+    <Stack horizontal styles={btnStackStyles} wrap horizontalAlign="space-between">
+      <HomeButton
+        theme={theme}
+        iconName={'Calendar'}
+        text={'Book an appointment'}
+        onClick={() => window.location.assign('/book')}
+      />
+      <HomeButton theme={theme} iconName={'Video'} text={'Start as Presenter'} onClick={callCreateRoom} />
+      <HomeButton
+        theme={theme}
+        iconName={'Link'}
+        text={'Join from link'}
+        onClick={() => window.location.assign('/visit')}
+      />
+    </Stack>
+  );
+};
+
+const HomeButton = ({ theme, iconName, text, onClick }): JSX.Element => {
+  const buttonStyles = getButtonStyles(theme);
+  return (
+    <DefaultButton styles={buttonStyles} iconProps={{ iconName }} onClick={onClick}>
+      <Text styles={buttonTextStyles}>{text}</Text>
+    </DefaultButton>
+  );
+};
+
+const LearnMore = (): JSX.Element => {
```

Comment:
Better to move this to  ""https://github.com/Azure-Samples/communication-services-virtual-visits-js/blob/main/client/src/components/LearnMoreItem.tsx" I think and rename it to LearnMore.tsx. So related components will be together in a file, and HomeComponent.tsx will be shorter.